src/k8s-extension/azext_k8s_extension/partner_extensions/ContainerInsights.py

```diff
@@ -495,6 +495,14 @@ def _get_container_insights_settings(cmd, cluster_resource_group_name, cluster_r
                 namspaces = dataCollectionSettings["namespaces"]
                 if isinstance(namspaces, list) is False:
                     raise InvalidArgumentValueError('namespaces must be an array type')
+            if 'enableContainerLogV2' in dataCollectionSettings.keys():
+                enableContainerLogV2Value = dataCollectionSettings["enableContainerLogV2"]
+                if not isinstance(enableContainerLogV2Value, bool):
+                    raise InvalidArgumentValueError('enableContainerLogV2Value value MUST be either True or False')
+            if 'streams' in dataCollectionSettings.keys():

```

Comment:
two missing changes

1) add default value i.e. streams = [  "Microsoft-ContainerInsights-Group-Default"]
2) update from this 
                              "streams" : [
                                "Microsoft-ContainerInsights-Group-Default"
                            ]
to 
"streams": streams

in ContainerInsights.py file.

Reply:
![image](https://github.com/Azure/azure-cli-extensions/assets/25573844/6cacbcc0-ce81-4d09-bb94-d22d23f8a635)
ContainerInsights.py is for ARC change, the aks-preview is using the code block in cli

Reply:
For Arc k8s-extension, you would need to make PR against this fork https://github.com/AzureArcForKubernetes/azure-cli-extensions and branch k8s-extension/public.  Arc k8s extension will merge the changes to main branch.